framework/docs/source/how-to-use-json-output.rst

```diff
@@ -0,0 +1,166 @@
+
+.. _json_error_output:
+
+.. code-block:: bash
+
+    $ flwr run --format json
+    {
+      "success": false,
+      "error-message": "Loading project configuration... \nSuccess\n There is no `[missing]` federation declared in the `pyproject.toml`.\n The following federations were found:\n\nfed-existing-1\nfed-existing-2\n\n"
+    }
+
+``flwr ls`` JSON output
+-----------------------
+
+The |flwr_ls|_ command lists all the runs in the current project. By default, the
+command list the details of one provided run ID or all runs in a Flower federation in a
+tabular format:
+
+.. code-block:: bash
+
+    $ flwr ls
+    Loading project configuration...
+    Success
+    📄 Listing all runs...
+    ┏━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━━━━┓
+    ┃    Run ID    ┃     FAB      ┃    Status    ┃ Elapsed  ┃  Created At  ┃  Running At  ┃ Finished At ┃
+    ┡━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━━━━━┩
+    │ 185995311804 │ flwrlabs/my… │ finished:co… │ 00:00:55 │ 2024-12-16   │ 2024-12-16   │ 2024-12-16  │
+    │ 1441032      │ (v1.0.0)     │              │          │ 11:12:33Z    │ 11:12:33Z    │ 11:13:29Z   │
+    ├──────────────┼──────────────┼──────────────┼──────────┼──────────────┼──────────────┼─────────────┤
+    │ 142007406570 │ flwrlabs/my… │ running      │ 00:00:05 │ 2024-12-16   │ 2024-12-16   │ N/A         │
+    │ 11601420     │ (v1.0.0)     │              │          │ 12:18:39Z    │ 12:18:39Z    │             │
+    └──────────────┴──────────────┴──────────────┴──────────┴──────────────┴──────────────┴─────────────┘
+
+Similar to the ``flwr run`` command, to get the output in JSON format, simply pass the
+``--format json`` flag:
+
+.. code-block:: bash
+
+    $ flwr ls --format json
+    {
+      "success": true,
+      "runs": [
+        {
+          "run-id": 1859953118041441032,
+          "fab-id": "flwrlabs/myawesomeapp1",
+          "fab-name": "myawesomeapp1",
+          "fab-version": "1.0.0",
+          "fab-hash": "014c8eb3",
+          "status": "finished:completed",
+          "elapsed": "00:00:55",
+          "created-at": "2024-12-16 11:12:33Z",
+          "running-at": "2024-12-16 11:12:33Z",
+          "finished-at": "2024-12-16 11:13:29Z"
+        },
+        {
+          "run-id": 14200740657011601420,
+          "fab-id": "flwrlabs/myawesomeapp2",
+          "fab-name": "myawesomeapp2",
+          "fab-version": "1.0.0",
+          "fab-hash": "014c8eb3",
+          "status": "running",
+          "elapsed": "00:00:09",
+          "created-at": "2024-12-16 12:18:39Z",
+          "running-at": "2024-12-16 12:18:39Z",
+          "finished-at": "N/A"
+        },
+      ]
+    }
+
+If the command fails, the JSON output will return two fields, ``success`` and
+``error-message``, as shown in the :ref:`failed command <json_error_output>` example.
```

Comment:
worth indicating that the ``error-message`` isn't the same as the one shown earlier (which is specific to `flwr run`)? Or maybe rephrasing this mini paragraph slightly is enough. This comment also applies to the `flwr stop` setting below.